packages/discord.js/src/structures/Message.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const { createComponent, Embed } = require('@discordjs/builders');
+const { Embed } = require('@discordjs/builders');
```

Comment:
This should probably import `'./embed'` instead.

Reply:
Fixed